crates/runtime_isolate/src/lib.rs

```diff
@@ -197,6 +208,7 @@ impl Isolate {
                 lines: 0,
                 requests_count: 0,
                 log_sender: options.log_sender.clone(),
+                compression_table: BTreeMap::<String, CompressionInner>::new(),
```

Comment:
What's the reason to use a `BTreeMap` over an `HashMap`?

Reply:
done